src/composite.comp

```diff
@@ -40,7 +64,7 @@ vec4 sampleLayer(sampler2D layerSampler, uint layerIdx, vec2 uv) {
 
 vec4 sampleLayer(uint layerIdx, vec2 uv) {
     if ((c_ycbcrMask & (1 << layerIdx)) != 0)
-        return sampleLayer(s_ycbcr_samplers[layerIdx], layerIdx, uv);
+        return srgbToLinear(sampleLayer(s_ycbcr_samplers[layerIdx], layerIdx, uv));
```

Comment:
Is the YCbCr sampler returning sRGB values or linear values? It'd be a bit nicer if we got linear values if Vulkan allows it.

Reply:
Previously it was going straight through, and we were presenting that so I imagine SRGB values. There is no _SRGB variant that for the format for a view that I can see.

With this it goes srgbToLinear -> blend -> linearToSrgb so that should be identical to what we were doing before.

Reply:
It's using `nir_convert_ycbcr_to_rgb` under the hood… It seems like this is returning sRGB values, so yeah need to decode that as done here.